__docs__/wonder-blocks-pill/pill-testing-snapshots.stories.tsx

```diff
@@ -107,7 +107,7 @@ export const Sizes: Story = {
                                     kind={kind}
                                     key={index}
                                 >
-                                    {kind}, {props.size}
+                                    {`${kind}, ${props.size}`}
```

Comment:
This updates the `children` prop so it is treated as a string, so we can see the typography styles applied by the component based on the size

https://github.com/Khan/wonder-blocks/blob/345fc42a49ca27b4748e9ec2b9a99eed781ee11b/packages/wonder-blocks-pill/src/components/pill.tsx#L80-L107